public/mushrooms.json

```diff
@@ -44,7 +48,8 @@
             "psychoactive": false,
             "poisonous": false,
             "deadly": false,
-            "cap": ["convex", "flat"],
+            "cap": ["convex", "flat", "Pure white or sometimes with grayish to brown or cinnamon fibrils"],
```

Comment:
Same here about the sentence. We do lookups on values like `convex` and `flat`, so a sentence wouldn't work with that.